### DISCUSSION TITLE
Sunshine breaks internet connection after 5 minutes of streaming. (Linux)

### DISCUSSION THREAD

**Discussion by Caddy8970** [LizardByte/community#539] [Sunshine] [answered]
### Select Topic Area

Bug

### Body

When I start a game from my phone via Moonlight, after 5 minutes of playing, Sunshine breaks the internet connection on the entire system, meaning that my PC is permanently offline until I reboot it, and ends the stream with error code -1.

I've tried to search the error online, but I couldn't find a fix or even a related discussion, so I think I might be the first one to have this exact issue.

I've also tried launching Sunshine with verbose logging, but the only errors it printed were related to no internet being available and not about breaking anything.

I'm currently using Debian 12. Will provide more info if necessary

**Comment by willow-sprite** [comment, marked as answer]
I've had a similar problem with sunshine taking my modem's 5G channel out of commission, probably just overwhelmed the old thing.

your issue could be the same or it could be the pc's network card getting overwhelmed and dropping the connection rather than the modem, or it could be something else.

possible solutions:
* if modem is the problem

a) use phone or pc as hotspot to prevent sunshine traffic going through modem.
- downside: for my network card, this gives me a range of ~2 rooms before cutting out or dropping frames.

b) use lower graphics settings
- downside: lower graphics settings

c) buy a newer modem

d) try using an ethernet cable, if it's the modem getting overwhelmed by traffic on one band, just using ethernet might help.

* if network card is the problem (which it probably isn't if your setup is good enough to run sunshine)

a) upgrade motherboard
downside: not worth the trouble if you aren't upgrading other stuff

b) buy a pcie wifi card
downside: requires an easy to access pcie port that isn't being used for something else.

c) buy a usb network card
downside: not as good as pcie
downside: requires a free usb port

[a comment by an automated account is not shown]
